src/main/lua/exasol/rls/RlsAdapter.lua

```diff
@@ -1,12 +1,12 @@
-local AbstractVirtualSchemaAdapter = require("exasolvs.AbstractVirtualSchemaAdapter")
-local adapter_capabilities = require("exasolrls.adapter_capabilities")
-local QueryRewriter = require("exasolrls.QueryRewriter")
-
 -- Derive from AbstractVirtualSchemaAdapter
 local RlsAdapter = {}
 RlsAdapter.__index = RlsAdapter
+local AbstractVirtualSchemaAdapter = require("exasol.vscl.AbstractVirtualSchemaAdapter")
 setmetatable(RlsAdapter, {__index = AbstractVirtualSchemaAdapter})
-local VERSION <const> = "1.3.0"
+local VERSION <const> = "1.4.0"
```

Comment:
Is there a test that verifies consistent version numbers here, in rockspec and in pom?